Models/PaymentTypes.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class PaymentTypes
+    {
+        [Key]
+        public int Id { get; set; }
+
+
+
+        [Required]
+        public int CustomersId { get; set; }
+        public Customers Customer { get; set; }
+        public string Name { get; set; }
+        public int? AccountNumber { get; set; }
```

Comment:
I believe the AccountNumber is actually required as a property on this class, is it not? (Discuss with your team as to why it might or might not be)